starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +23,49 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord(words) {
+  var longestWord = '';
+  if (words.length > 0) {
+    for (var i = 0; i < words.length; i++) {
+    if (words[i].length > longestWord.length) {
+       longestWord = words[i];
+    }
+  }
+  return longestWord;
+  } else {
+  return undefined;
+  }
+}
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumArray(numbers) {
+  var sum = 0;
+  for (var i = 0;i < numbers.length; i++){
+   sum += numbers[i];
+  }
+  return sum;
+}
+
 // Calculate the Average
 
 var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function averageNumbers(numbersAvg) {
+  var sum = 0;
+  if(numbersAvg.length > 0){
+  for (var i = 0;i < numbersAvg.length; i++){
```

Comment:
Have you done this before? Instead of repeating the code, what about reusing the function?
```
function averageNumbers(numbersAvg) {
  var sum = 0;
  if(numbersAvg.length > 0) {
    return sumArray(numbersAvg) / numbersAvg.length;
  } else{
    return undefined;
  }
}
```